lib/dolla/end_points.rb

```diff
@@ -0,0 +1,30 @@
+module Dolla
+  module EndPoints
+
+    XML_NAMESPACE = {
+      :default  => 'http://webservice.interred.adquira.com.mx',
```

Comment:
Now that you are porting everything to this gem, it'd be nice to rewrite all this hash definitions replacing old hash rocket style. `default: 'something'` instead of `default => 'something'`